services/database/user.ts

```diff
@@ -35,4 +40,100 @@ async function getUsers() {
     return users;
 }
 
-export { getUser, getUsers };
+/* 
+TODO (6/27/21): https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#create 
+doc does not specify what happens if the create operation fails - does that mean it should always
```

Comment:
I believe the create operation can fail if the required fields aren't specified. It just won't create a record if it fails.

Reply:
Not sure if that's what you're asking